libs/ha/ha_common_libs_k8s.py

```diff
@@ -679,12 +678,12 @@ def create_bucket_copy_obj(event, s3_test_obj=None, bucket_name=None, object_nam
         LOGGER.info("Start copy object to buckets: %s", list(bkt_obj_dict.keys()))
         for bkt_name, obj_name in bkt_obj_dict.items():
             try:
-                status, response = s3_test_obj.copy_object(source_bucket=bucket_name,
-                                                           source_object=object_name,
-                                                           dest_bucket=bkt_name,
-                                                           dest_object=obj_name)
-                LOGGER.info("Response: %s", response)
-                copy_etag = response['CopyObjectResult']['ETag']
+                response = s3_test_obj.copy_object(source_bucket=bucket_name,
+                                                   source_object=object_name,
+                                                   dest_bucket=bkt_name,
+                                                   dest_object=obj_name)
+                LOGGER.info("Response: %s", response[1])
+                copy_etag = response[1]['CopyObjectResult']['ETag']
```

Comment:
Is this change not affecting the existing test cases?

Reply:
Yes. There was one unused variable warning. And here we are just need of 2 element of function return value. Logic is same and its not major change